packages/indexer-agent/src/agent.ts

```diff
@@ -1017,10 +1054,12 @@ class Agent {
       poi === null ||
       poi === utils.hexlify(Array(32).fill(0))
     ) {
+      const tempPoi = await this.resolvePOI(epochStartBlock, existingAllocation)
```

Comment:
should agent be able to reallocate to failed subgraphs? could set poi to this temp poi to reallocate (the current behavior is to keep failed subgraphs open)